pydantic/json_schema.py

```diff
@@ -1630,6 +1630,17 @@ def url_schema(self, schema: core_schema.UrlSchema) -> JsonSchemaValue:
         self.update_with_validations(json_schema, schema, self.ValidationsMapping.string)
         return json_schema
 
+    def uuid_schema(self, schema: core_schema.UuidSchema) -> JsonSchemaValue:
+        """Generates a JSON schema that matches a schema that defines a UUID.
+
+        Args:
+            schema: The core schema.
+
+        Returns:
+            The generated JSON schema.
+        """
+        return {'type': 'string', 'format': 'uuid'}
```

Comment:
I think we might want to have the uuid version in the format (where applicable)?

Reply:
I think we deduplicate the information because if we have a uuid4 the json is : ` {'type': 'string', 'format': 'uuid4'}`